bundles/org.openhab.binding.danfossairunit/src/main/java/org/openhab/binding/danfossairunit/internal/DanfossAirUnitHandler.java

```diff
@@ -86,56 +87,55 @@ public void initialize() {
         valueCache = new ValueCache(config.updateUnchangedValuesEveryMillis);
         try {
             hrv = new DanfossAirUnit(InetAddress.getByName(config.host), 30046);
+            DanfossAirUnit danfossAirUnit = hrv;
```

Comment:
By assigning the non-null value the local variable first it will make the null checker happy.
```suggestion
            DanfossAirUnit danfossAirUnit = new DanfossAirUnit(InetAddress.getByName(config.host), 30046);
            hrv = danfossAirUnit;
```

Reply:
I did the changes after seeing that the null checker was happy, trust me.